pkg/ddevapp/drupal.go

```diff
@@ -247,3 +325,25 @@ func isDrupal8App(app *DdevApp) bool {
 	}
 	return false
 }
+
+// isDrupal6App returns true if the app is of type Drupal6
+func isDrupal6App(app *DdevApp) bool {
+	if _, err := os.Stat(filepath.Join(app.AppRoot, app.Docroot, "misc/ahah.js")); err == nil {
+		return true
+	}
+	return false
+}
+
+// drupal7ConfigOverrideAction overrides php_version for D6, since it is incompatible
+// with php7+
+func drupal7ConfigOverrideAction(app *DdevApp) error {
+	app.PHPVersion = "7.0"
```

Comment:
Do we need an override for D7? I think it should work okay on PHP 7.1.

If we definitely need the override, the comment probably needs updated.

Reply:
@cweagans That was definitely a gimme. I was trying to differentiate and threw that in. I was kind of thinking it would make ddev work longer in the future with d7 (when breaks do come). But it may have been silly. I also wanted something more to test against.